services/movie_session.py

```diff
@@ -0,0 +1,51 @@
+from django.db.models import QuerySet
+from datetime import datetime
+
+from db.models import MovieSession
+
+
+def create_movie_session(
+    movie_show_time: datetime,
+    movie_id: int,
+    cinema_hall_id: int
+) -> MovieSession:
+    return MovieSession.objects.create(
+        show_time=movie_show_time,
+        movie_id=movie_id,
+        cinema_hall_id=cinema_hall_id
+    )
+
+
+def get_movies_sessions(
+    session_date: datetime = None
```

Comment:
The `session_date` parameter should be of type `date` instead of `datetime` to correctly filter sessions by date. Consider changing the parameter type to `date`.